tests/x11/kontact.pm

```diff
@@ -35,12 +35,23 @@ sub run {
         # KF5-based account assistant ignores alt-f4
         wait_screen_change { send_key 'alt-c' } if match_has_tag('test-kontact-1');
     } until (match_has_tag('kontact-window'));
-    assert_and_click 'close_kontact';
+    assert_screen [qw(kontact-error close_kontact)];
+
+    # workaround for fatal error bsc#1105207
+    if (match_has_tag('kontact-error')) {
+        record_soft_failure('encounter fatal error, bsc#1105207') && return;
+    }
+    else {
+        assert_and_click 'close_kontact';
+    }
     # Since gcc7 used for packages within openSUSE Factory kontact seems to
     # persist consistently as a process in the background causing kontact to
     # be "restored" after a re-login/reboot causing later tests to fail. To
     # prevent this we explicitly stop the kontact background process.
     x11_start_program('killall kontact', valid => 0);
+    # sometimes 'killall kontact' doesn't help further, so check whether
+    # akregator got closed. If not, try with record_soft_failure and return
+    record_soft_failure('akregator cannot be closed, related to issue of bsc#1105207') if (check_screen 'akregator-not-closed');
```

Comment:
I don't see how akregator is related to https://bugzilla.suse.com/show_bug.cgi?id=1105207 . Can we please split the concerns and you just focus on the kontact error and not akregator?

For easier testing have you considered https://progress.opensuse.org/issues/35589#note-35 to just schedule booting from an image and call kontact? If you not schedule "amarok" it should prevent the "akregator" problem

Reply:
updated now